ipyvolume/pylab.py

```diff
@@ -422,11 +431,17 @@ def quiver(x, y, z, u, v, w, size=default_size * 10, size_selected=default_size_
     """
     fig = gcf()
     _grow_limits(x, y, z)
-    scatter = ipv.Scatter(x=x, y=y, z=z, vx=u, vy=v, vz=w,
-                             color=color, size=size, color_selected=color_selected, size_selected=size_selected,
-                             geo=marker, **kwargs)
-    fig.scatters = fig.scatters + [scatter]
-    return scatter
+    if 'geo' in kwargs:
+        marker = kwargs['geo']
+        del kwargs['geo']
+    if 'vx' in kwargs or 'vy' in kwargs or 'vz' in kwargs:
+        raise KeyError('Please use u, v, w instead of vx, vy, vz')
```

Comment:
I don't think this should be added, quiver had never vx etc as arguments, is we were doing some backward incompatible tricks this would make sense though.

Reply:
same reasoning as before:

`quiver(..., u=u, ..., vx=u)` would be legal but bad.  this just raises an exception to notify the user. all of this stems from the fact that any `kwargs` eventually gets passed to the lowest level scatter call.

Reply:
good point, indeed!